apex/transformer/testing/distributed_test_base.py

```diff
@@ -58,3 +67,50 @@ def _run(cls, rank, test_name, file_name, pipe):
 
         dist.destroy_process_group()
         sys.exit(0)
+
+    def _setup_pre_spawn(self):
+        pass
+
+
+class NcclDistributedTestBase(DistributedTestBase):
+
+    DISTRIBUTED_BACKEND = "nccl"
+
+
+@unittest.skipUnless(
+    HAS_TORCH_UCC,
+    "Requires [`torch_ucc`](https://github.com/facebookresearch/torch_ucc)",
+)
```

Comment:
Ideally I want to check driver version as UCC requires >=470.42.01

Reply:
resolved in https://github.com/NVIDIA/apex/pull/1380/commits/813bc451b8384f847f9b4ed27e4884c2bdc62b94